### PR TITLE
Capitalize markdown

### PR DESCRIPTION
**Pull request by glenn-jocher**
## 🛠️ PR Summary

<sub>Made with ❤️ by [Ultralytics Actions](https://www.ultralytics.com/actions)</sub>

### 🌟 Summary
Minor refactor and documentation polish: convert several helper methods to `@staticmethod`, tidy docs formatting, and apply small JS/CSS cleanups 🧹✨

### 📊 Key Changes
- Converted multiple internal helper methods across the codebase to `@staticmethod` (e.g., in augmentation, tuner, SAM predictors, YOLO detect predictor, NN blocks/heads, git utils, tqdm utils) 🔧
- Small SAM predictor state init cleanup: initializes `text_prompt` and `per_frame_geometric_prompt` directly in the state dict for clarity 🧠
- Docs improvements:
  - Fixed step numbering in the YOLOv7 ONNX usage guide (0→1, 1→2, etc.) 📚✅
  - Standardized “Markdown” capitalization in docstrings/comments 📝
- Frontend tweaks for docs site:
  - JS: replaced `var` with `const` in `giscus.js` for modern style and safer scoping 🌐
  - CSS: simplified `padding: 40px 0px;` → `padding: 40px 0;` 🎨
- Export utility cleanup: simplified tuple returns in IMX export helpers (`return *x, ...` instead of `return (*x, ...)`) 📦

### 🎯 Purpose & Impact
- Improves code clarity and maintainability by marking methods that don’t use instance state as static, reducing confusion and easing future refactors 🛠️
- Slightly cleaner and more consistent docs (especially the YOLOv7 ONNX instructions), reducing user friction when following setup steps 📖🚀
- Minor docs-site code/style modernizations that reduce linting/style issues and improve consistency, with effectively no behavior change for end users ⚙️✅
- Overall: a low-risk polish PR—mostly internal cleanup and documentation correctness, with minimal likelihood of breaking changes 🧩👍

### CODE REVIEW

**Comment by UltralyticsAssistant** [comment]
👋 Hello @glenn-jocher, thank you for submitting a `ultralytics/ultralytics` 🚀 PR! This is an automated message—an engineer will review and assist soon 🛠️✨ To ensure a seamless integration of your work, please review the following checklist:

- ✅ **Define a Purpose**: Clearly explain the purpose of your fix or feature in your PR description, and link to any [relevant issues](https://github.com/ultralytics/ultralytics/issues). Ensure your commit messages are clear, concise, and adhere to the project's conventions.
- ✅ **Synchronize with Source**: Confirm your PR is synchronized with the `ultralytics/ultralytics` `main` branch. If it's behind, update it by clicking the 'Update branch' button or by running `git pull` and `git merge main` locally.
- ✅ **Ensure CI Checks Pass**: Verify all Ultralytics [Continuous Integration (CI)](https://docs.ultralytics.com/help/CI/) checks are passing. If any checks fail, please address the issues.
- ✅ **Update Documentation**: Update the relevant [documentation](https://docs.ultralytics.com/) for any new or modified features.
- ✅ **Add Tests**: If applicable, include or update tests to cover your changes, and confirm that all tests are passing.
- ✅ **Sign the CLA**: Please ensure you have signed our [Contributor License Agreement](https://docs.ultralytics.com/help/CLA/) if this is your first Ultralytics PR by writing "I have read the CLA Document and I sign the CLA" in a new message.
- ✅ **Minimize Changes**: Limit your changes to the **minimum** necessary for your bug fix or feature addition. _"It is not daily increase but daily decrease, hack away the unessential. The closer to the source, the less wastage there is."_  — Bruce Lee

For more guidance, please refer to our [Contributing Guide](https://docs.ultralytics.com/help/contributing/). Don't hesitate to leave a comment if you have any questions. Thank you for contributing to Ultralytics! 🚀

File: ultralytics/utils/export/imx.py [hunk 1]
```diff
@@ -116,7 +116,7 @@ def pose_forward(self, x: list[torch.Tensor]) -> tuple[torch.Tensor, torch.Tenso
     kpt = torch.cat([self.cv4[i](x[i]).view(bs, self.nk, -1) for i in range(self.nl)], -1)  # (bs, 17*3, h*w)
     x = Detect.forward(self, x)
     pred_kpt = self.kpts_decode(bs, kpt)
-    return (*x, pred_kpt.permute(0, 2, 1))
+    return *x, pred_kpt.permute(0, 2, 1)
```
Comment (UltralyticsAssistant) [inline ultralytics/utils/export/imx.py:119]: ❗ **CRITICAL**: This change introduces invalid Python syntax. `return *x, ...` is not valid in a return statement (star-unpacking requires parentheses/a tuple/list). This will raise a `SyntaxError` at import time.

**Suggested change:**
```suggestion
    return (*x, pred_kpt.permute(0, 2, 1))
```

File: ultralytics/utils/export/imx.py [hunk 2]
```diff
@@ -125,7 +125,7 @@ def segment_forward(self, x: list[torch.Tensor]) -> tuple[torch.Tensor, torch.Te
     bs = p.shape[0]  # batch size
     mc = torch.cat([self.cv4[i](x[i]).view(bs, self.nm, -1) for i in range(self.nl)], 2)  # mask coefficients
     x = Detect.forward(self, x)
-    return (*x, mc.transpose(1, 2), p)
+    return *x, mc.transpose(1, 2), p
```
Comment (UltralyticsAssistant) [inline ultralytics/utils/export/imx.py:128]: ❗ **CRITICAL**: Same issue here: `return *x, ...` is invalid syntax and will break module import/execution.

**Suggested change:**
```suggestion
    return (*x, mc.transpose(1, 2), p)
```

**Review by UltralyticsAssistant — COMMENTED** [review]
## 🔍 PR Review

<sub>Made with ❤️ by [Ultralytics Actions](https://www.ultralytics.com/actions)</sub>

PR is generally clean (docs capitalization, minor JS/CSS tweaks, and @staticmethod refactors look reasonable). One critical blocker: `ultralytics/utils/export/imx.py` now contains invalid Python syntax (`return *x, ...`) in two places, which will cause a SyntaxError on import. Re-wrapping these returns in parentheses fixes it.

💬 Posted 2 inline comments

**Comment by glenn-jocher** [comment]
## ⚡ Actions Trigger

<sub>Made with ❤️ by [Ultralytics Actions](https://www.ultralytics.com/actions)</sub>

GitHub Actions below triggered via workflow dispatch for this PR at 2025-12-25 13:49:22 UTC with `@ultralytics/run-ci` command:

* ✅ [CI](https://github.com/ultralytics/ultralytics/actions/runs/20506058231): `ci.yml` run 15849
* ✅ [Publish Docker Images](https://github.com/ultralytics/ultralytics/actions/runs/20506058543): `docker.yml` run 1554

**Comment by UltralyticsAssistant** [comment]
Merged—thank you @glenn-jocher! 🚀

“Perfection is achieved, not when there is nothing more to add, but when there is nothing left to take away.” — **Antoine de Saint-Exupéry**

This PR is a great example of that mindset: converting helpers to `@staticmethod`, tightening predictor state init, and polishing docs + small JS/CSS cleanups all reduce ambiguity and friction without changing behavior. These “low-risk” refinements compound into a codebase that’s easier to maintain and a smoother experience for everyone building with Ultralytics and YOLO11.
